core/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://java.sun.com/xml/ns/javaee" 
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+       xmlns:weld="http://jboss.org/schema/weld/beans" 
+       xsi:schemaLocation="
+          http://java.sun.com/xml/ns/javaee http://docs.jboss.org/cdi/beans_1_0.xsd
+          http://jboss.org/schema/weld/beans http://jboss.org/schema/weld/beans_1_1.xsd">
+    <weld:scan>
+        <!-- Don't let Arquillian choke on MR-Jars -->
```

Comment:
Hmm this should be fixed in more recent version of the container - https://github.com/arquillian/arquillian-container-weld/commit/7296f97caac5ceaf8a5ec3a11a483a7c36e84a44#diff-797455798ac6dee103ff0fd610638bb5
Version of arq container seems to come from SR parent and newer versions are already updated, just bump it and you can remove this file :)

Reply:
Actually there is already https://github.com/smallrye/smallrye-context-propagation/pull/181
So you can choose between merging and rebasing or updating it as part of this PR, up to you.

Reply:
Well spotted, I've merged #181, rebased and removed that file.